go.mod

```diff
@@ -53,8 +53,8 @@ require (
 
 replace github.com/gogo/protobuf => github.com/ElrondNetwork/protobuf v1.3.2
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.35 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.37
+replace github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.38 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.38
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.35 => github.com/ElrondNetwork/arwen-wasm-vm v1.3.37
+replace github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.38 => github.com/ElrondNetwork/arwen-wasm-vm v1.3.38
 
-replace github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.42 => github.com/ElrondNetwork/arwen-wasm-vm v1.4.42
+replace github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.42 => github.com/ElrondNetwork/arwen-wasm-vm v1.4.44-0.20220308121737-37aae3950f1b
```

Comment:
proper release + change the v1.4.42 to v1.4.44

Reply:
Done.